src/Button/index.jsx

```diff
@@ -1,8 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
 import Button from 'react-bootstrap/Button';
 import ButtonDeprecated from './deprecated';
 
-Button.Deprecated = ButtonDeprecated;
+import Icon from '../Icon';
 
-export default Button;
+const WrappedButton = React.forwardRef(({
+  children,
+  iconAfter,
+  iconBefore,
+  ...props
+}, ref) => (
+  <Button
+    {...props}
+    className={classNames(props.className)}
+    ref={ref}
+  >
+    <span className={classNames('d-flex', 'align-items-center')}>
```

Comment:
Please avoid using css utility classes inside paragon components and instead prefer to update the Button.scss in this directory. Utility classes add `!important` directives to styles and so should be used sparingly within Paragon itself.

Reply:
Fixed! @abutterworth